### DISCUSSION TITLE
Welcome to whitepaper Discussions!

### DISCUSSION THREAD

**Discussion by miladsoft** [miladsoft/whitepaper#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dangershony** [comment]
This is excellent, I am very happy to see the initiative of making a white paper.

I was part of the team that built blockcore.

I started at Stratis just after the ICO where I was the main Dev and later CTO, we build a Bitcoin fullnode and then adapted it to Stratis and it's POS algorithm (which is based on blackcoin and poscoin which is an early fork of litecoin but with POS, litecoin itself was an early fork of Bitcoin) and later build sidechain technology and created Cirrus the stratis sidechain which is a hybrid of a utxo model but adapted to ethereum style VM but the engine is the C# runtime (or simply put smart contracts in C#)

Later I was part of a community effort to build blockcore, we forked the Stratis code base removed the Cirrus code and left only the utxo model, we did that to support the other chains that build on top of the fullnode codebase.

We also introduced POSv4 for the XDS chain.
